Fix default argument warnings from clang-tidy

mfasDa opened the pull request:
- ShishKebapTrd1Module: Set default arguments common
  for both constructors in header file and remove them
  from the constructor implementation
- Spaceframe: All members were never re-assinged, they
  can actually be constants. Converted to constants,
  removed custom implementation for assignment
  operator and copy constructor, and declared
  constructor default

mfasDa commented:
Related to the warnings from o2checkcode from EMCAL classes in #1729

matthiasrichter commented:
Hi Markus, can you apply the formatting patch from travis or use `git clang-format HEAD~1` on this branch. Thanks.

mfasDa commented:
Hi @matthiasrichter ,

Sorry, I overlooked the format issues. I fixed them and rebased them into the original commit -  it should be fine now.

Cheers

Markus
